app-preprod.yaml

```diff
@@ -34,9 +34,7 @@ spec:
     requests:
       memory: 1024Mi
       cpu: 200m
-  ingresses: # Optional. List of ingress URLs that will route HTTP traffic to the application.
-    - https://familie-ba-mottak.nais.preprod.local #deprecated
-    - https://familie-ba-mottak.dev-fss-pub.nais.io
```

Comment:
Trengs ikke pub adressen for prosessering?

Reply:
Her blingset jeg og slettet feil ingress